2-mandatory/1-writers.js

```diff
@@ -59,60 +59,76 @@ Exercise 1:
   and insert the corresponding values to the place holders that are indicated in curly braces:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
-function logAllWriters() {
-  // write your code to log all writers here
+function logAllWriters(obj) {
+   let newObj=  obj.map(x => {
+  return  console.log(
```

Comment:
You should not return the `console.log` inside the map, you should just return the string straight or assign it to a var and return the var